### ISSUE TITLE
Using 'mj-navbar' requires Google Fonts download for icons unnecessarily

### ISSUE THREAD

**Issue by rootwork** [mjmlio/mjml#2262]
**Describe the bug**
If you use `<mj-navbar>` in your email, the resulting HTML will include a download for the Google Font "Ubuntu" even if no fonts are specified elsewhere in the configuration.

It appears to me that this is in order to render the mobile navigation menu icons  ☰  and  ⊗ .

Downloading an entire Google Font, in multiple weights, just for two icons seems overkill. Suggested alternatives:

- rely on system fonts
- only download the two characters necessary (see [this Smashing Magazine article](https://www.smashingmagazine.com/2019/06/optimizing-google-fonts-performance/), item number 7, for the technique)
- provide an option to turn off this behavior
- at least disclose in the docs that using this element results in a font download

**To Reproduce**
1. Add `<mj-navbar>` somewhere in your MJML, for instance [this demo](https://mjml.io/try-it-live/Zzvara5GZhW).
2. View the HTML.
3. Search for "googleapis" in the HTML to see the two imports (in the demo it's lines 59-62).

**Expected behavior**

Using a particular component will not result in a font download.

**MJML environment (please complete the following information):**
 - OS: Linux Mint/Ubuntu 20.1
 - MJML Version: 4.9.1, 4.9.3
 - MJML tool used: mjml-cli, gulp-mjml

**Additional context**

There are a couple other small issues with `<mj-navbar>` if you're working on it anyway:

1. The `id` attribute for the checkbox-hack (`mj-menu-checkbox`) changes on every build, resulting in diff noise.
2. The icons that are inserted are actually in the text body of the email, meaning they'll show up if there's no preview text defined and no text appears before them, and if you do a separate HTML-to-text conversion they'll get included there as well. I'm not sure of an alternative for this, though, since email clients don't all support inline SVG, the `content:` CSS attribute, the `::before` or `::after` selectors (e.g. for creating CSS shapes), or inline base64 images. But it might be worth documenting that these icons are there in the text.

**Comment by iRyusa** [comment]
Every components have a default font family with Ubuntu and it can be
overriden with mj-attributes / mj-all. MJML will import few of them if they
are used in the document (
https://github.com/mjmlio/mjml/blob/master/packages/mjml-core/src/index.js#L93-L101
).

https://mjml.io/try-it-live/b6nil699zi0

Just reseting ico-font-family and font-family will remove that import.

1. mjml2html is not a pure function so it's intended to have a different id
on each render ( it allows to use multiple navbar and multiple checkbox
hack )
2. I don't know that we really need to document such a small thing,
inspecting the HTML will give you the same answer if you really need to
transform the produced HTML you will at some point read it

**Comment by rootwork** [comment]
I forgot to say that I had tried setting `font-family` on `mj-all` -- but I didn't realize it would also accept `ico-font-family`! That's great news.

Maybe it's just me being dense, but when you say "You can use mj-all to set default attributes for every components inside your MJML document" at https://documentation.mjml.io/#mj-attributes I didn't realize that meant _MJML_ default attributes. I just thought it was CSS attributes (like in the example given, setting the font-family to Arial). It might be worth adding an example like `ico-font-family` that makes it clear that you can set MJML attributes here as well.

I respect your positions on the other two items.

**Comment by iRyusa** [comment]
We can maybe rephrase a bit mj-attributes's documentation to ensure there's no confusion about MJML attributes, css attributes and mj-class 👍 
I'll try to do a PR about this
